tests/specs/check/typecheck_doc_duplicate_identifiers/__test__.jsonc

```diff
@@ -0,0 +1,5 @@
+{
+  "args": "check --doc --config ../../../config/deno.json mod.ts",
```

Comment:
Can you change this so that the spec test is self contained and doesn't traverse up ancestor directories?

Reply:
I referred to other existing spec tests like: https://github.com/denoland/deno/blob/c9065103b8ef0317fd8141f554fdc0a2f801e844/tests/specs/future/runtime_api/__test__.jsonc#L4

In any case I don't think we can get rid of relative path completely because we want to import @std modules from the git submodule which is located at `<repo_root>/tests/util/std`. Do you have any good idea to make this self-contained?

Reply:
That's a good catch. We need to find a better solution here

Reply:
We could have `deno.json` in each test directory that contains `importMap: "../../../../import_map.json"` to utilize [the import map](https://github.com/denoland/deno/blob/f2b53d42acf96044e56b21808cfc3f33a9501ec3/import_map.json) located at the repo root, but this still has relative path and thus not self-contained. Any other ideas?